src/user-research/recruiting-participants-and-protecting-privacy.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Recruiting research participants and protecting their privacy
+related_order: 120
+last_reviewed_at: ""
+redirect_from:
+  - /user-research/protecting-participant-privacy
+---
+
+Good user research depends on recruiting appropriate participants.
+
+This might mean recruiting:
+
+* members of the public with experience of a particular life event, like bereavement
+* small business owners who claim a particular tax relief, like an investment allowance
+* public servants who review and make decisions about applications, like those for visas
+
+We make sure that our recruitment practices are inclusive and accessible, so that we don’t exclude groups such as disabled people, and people with other protected characteristics.
+
+And we carefully manage participants’ contact details and other personal data to protect their privacy and comply with the law.
+
+Note that this guidance applies to all dxw staff, dxw friends, and partner and client staff doing research activities, like interviews, visits, surveys and tests.
+
+## Following our client’s practices
+
+When we are following our client’s practices in how we [manage research data](https://playbook.dxw.com/user-research/taking-care-of-research-data/), we will
```

Comment:
`clients'`?

Reply:
Hmmm. In this case I think we're talking about an individual client, rather than out clients collectively. So client's.